components/brave_wallet/browser/asset_ratio_response_parser.cc

```diff
@@ -301,4 +301,134 @@ mojom::BlockchainTokenPtr ParseTokenInfo(const std::string& json,
       *symbol, decimals, true, "", "", chain_id, coin);
 }
 
+bool ParseCoinMarkets(const std::string& json,
+                      std::vector<mojom::CoinMarketPtr>* values) {
+  DCHECK(values);
+  // {
+  //   "payload": [
+  //     {
+  //       "id": "bitcoin",
+  //       "symbol": "btc",
+  //       "name": "Bitcoin",
+  //       "image":
+  //       "https://assets.coingecko.com/coins/images/1/large/bitcoin.png?1547033579",
```

Comment:
```suggestion
  //       "https://assets.cgproxy.brave.com/coins/images/1/large/bitcoin.png?1547033579",
```

Reply:
@thypon This is a comment showing an example of  a response from coingecko.